spark/src/main/java/org/apache/zeppelin/spark/ZeppelinR.java

```diff
@@ -107,9 +108,11 @@ public Object getValue() {
    * @param rCmdPath R repl commandline path
    * @param libPath sparkr library path
    */
-  public ZeppelinR(String rCmdPath, String libPath, int sparkRBackendPort) {
+  public ZeppelinR(String rCmdPath, String libPath,
+                   int sparkRBackendPort, SparkVersion sparkVersion) {
```

Comment:
This is a nitpick (so no need to change if you do not feel like it) but Is that IDE formatting?
As [horizontal alignment](https://google.github.io/styleguide/javaguide.html#s4.6.3-horizontal-alignment) is usually discouraged.